### PR TITLE
Checking before adding attributes (Market-1501)

### PR DESCRIPTION
**Pull request by yasakova-anastasia**
### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md)
- [ ] I have updated the [documentation](
  https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have [linked related issues](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: datumaro/plugins/market1501_format.py [hunk 1]
```diff
@@ -88,10 +88,11 @@ def _load_items(self, rootdir):
...
 
-            attributes = item.attributes
-            attributes['query'] = subdir == Market1501Path.QUERY_DIR
-            attributes['person_id'] = pid
-            attributes['camera_id'] = camid
+            if pid != Market1501Path.UNKNOWN_ID:
+                attributes = item.attributes
+                attributes['query'] = subdir == Market1501Path.QUERY_DIR
+                attributes['person_id'] = pid
+                attributes['camera_id'] = camid
```
Comment (zhiltsov-max) [inline datumaro/plugins/market1501_format.py:95]: Why this information is not needed, when id is unknown?

Comment (yasakova-anastasia) [reply datumaro/plugins/market1501_format.py:95]: I think that we shouldn't include this information. If we initially didn't have attributes in the dataset, we shouldn't add them.

Comment (zhiltsov-max) [reply datumaro/plugins/market1501_format.py:95]: I think, it is impossible for us to know at this point, if these values were present or not. Also, how about the `test` subset?

The second question is why do we exclude all the attributes?

Comment (yasakova-anastasia) [reply datumaro/plugins/market1501_format.py:95]: For example, we cannot check 'query' like that, so we check only one and it decides for all.

> I think, it is impossible for us to know at this point, if these values were present or not. Also, how about the `test` subset?
What do you mean with `test` subset?

Comment (zhiltsov-max) [reply datumaro/plugins/market1501_format.py:95]: I mean the `bounding_box_test` directory from https://www.kaggle.com/pengcw1/market-1501/data, which contains images having -1, but camera ids and sequence ids are meaningful.

Comment (yasakova-anastasia) [reply datumaro/plugins/market1501_format.py:95]: Then we think we have no attributes when `person_id` and `camera_id` are -1.

**Review by zhiltsov-max — APPROVED** [review]
I'm still not convinced that this approach is right, but I suggest to merge and look.

**Comment by yasakova-anastasia** [comment]
> I'm still not convinced that this approach is right, but I suggest to merge and look.

Yes, I agree with you

**Pull request merged**